src/ostorlab/runtimes/local/agent_runtime.py

```diff
@@ -30,13 +30,6 @@
 
 MOUNT_VARIABLES = {"$CONFIG_HOME": str(configuration_manager.OSTORLAB_PRIVATE_DIR)}
 
-HEALTHCHECK_HOST = "0.0.0.0"
-HEALTHCHECK_PORT = 5000
-SECOND = 1000000000
-HEALTHCHECK_RETRIES = 300
-HEALTHCHECK_TIMEOUT = 30 * SECOND
-HEALTHCHECK_START_PERIOD = 1 * SECOND
-HEALTHCHECK_INTERVAL = 1 * SECOND
 MAX_SERVICE_NAME_LEN = 63
```

Comment:
You still need to remove the healthcheck from the agent to speed up its start time.